lib/generators/sufia/templates/migrations/change_featured_work_generic_file_id_to_generic_work_id.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeFeaturedWorkGenericFileIdToGenericWorkId < ActiveRecord::Migration
+  def change
+    return unless column_exists?(:featured_works, :generic_file_id)
+    rename_column :featured_works, :generic_file_id, :generic_work_id
```

Comment:
@awead :speech_balloon: 
> ... I've put it under the upgrade700 step, but since the engine cart is running this generator as well, we need to skip it since the table and its correct column already exists. ...

FWIW, this wouldn't be the only migration that has a conditional in it, so this solution is OK. We have a backlog ticket to work on spiffying up our migration-having generators, so we'll fix that up at some point. For now, :shipit: